client/state/selectors/get-user-devices.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { get, values } from 'lodash';
+
+/**
+ * Returns the current user's devices
+ *
+ * @param  {Object} state Global state tree
+ * @return {Object}       current user's devices
+ */
+export default function getUserDevices( state ) {
+	return values( get( state, 'userDevices.items' ) );
+}
```

Comment:
fun fact! we're just returning the evaluation of an expression so we have the opportunity to remove noisy syntax :smile:

```js
export const getUserDevices = state => values( get( state, 'userDevices.items' ) )
```